news/10693.trivial.rst

```diff
@@ -0,0 +1 @@
+Replace all our tox operations with Nox.
```

Comment:
Change this to process? I think this is significant enough to call out. :)

Reply:
Do you mean to rename the news to `10693.process.rst`? Or what do you mean?

Reply:
Yup, exactly that. Sorry for being unclear. 😅

Reply:
Don't worry. I didn't know a `.process.rst` were possible...